site/src/components/introduction/building-a-chart-complete.js

```diff
@@ -0,0 +1,87 @@
+// a random number generator
+var generator = fc.data.random.walk()
+    .steps(11);
+
+var valueformatter = d3.format('$f');
+var dateFormatter = d3.time.format('%b');
+
+// create the data
+var data = {
+    // target values for the annotations
+    targets: [{
+        name: 'low',
+        value: 8.5
+    }, {
+        name: 'high',
+        value: 11.2
+    }],
+    // randomly generated sales data
+    sales: generator(5).map(function(d, i) {
+        return {
+            month: dateFormatter(new Date(0, i + 1, 0)),
+            sales: d + i / 2
+        };
+    })
+};
+
+var yExtent = fc.util.extent()
+    .include(0)
+    .pad([0, 0.5])
+    .fields('sales');
+
+var chart = fc.chart.cartesian(
+        d3.scale.ordinal(),
+        d3.scale.linear())
+    .chartLabel('2015 Cummulative Sales')
+    .margin({top: 30, right: 45, bottom: 40})
+    .xDomain(data.sales.map(function(d) { return d.month; }))
+    .yDomain(yExtent(data.sales))
+    .yTicks(5)
+    .yTickFormat(valueformatter)
+    .yLabel('Sales (millions)')
+    .yNice();
+
+var bar = fc.series.bar()
+    .xValue(function(d) { return d.month; })
+    .yValue(function(d) { return d.sales; })
+    .decorate(function(selection) {
+        selection.select('path')
```

Comment:
I'd possibly select `.bar > path` to make it even clearer what's being selected.